packages/core/src/App/Containers/Layout/header/traders-hub-onboarding.tsx

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import { useHistory } from 'react-router-dom';
-import { useLocalStorage } from 'usehooks-ts';
```

Comment:
no more local storage in use to share state between random parts of application, yay :D